static/js/settings_linkifiers.js

```diff
@@ -40,6 +43,22 @@ function sort_url(a, b) {
     return compare_values(a.url_format, b.url_format);
 }
 
+function open_linkifier_edit_form(linkifier_id) {
+    const linkifiers_list = page_params.realm_linkifiers;
+    const linkifier = linkifiers_list.find((linkifier) => linkifier.id === linkifier_id);
+    const html = render_admin_linkifier_edit_form({
+        linkifier_id,
+        pattern: linkifier.pattern,
+        url_format_string: linkifier.url_format,
+    });
+    const div = $(html);
```

Comment:
Let's rename `html` to `edit_form_html` and perhaps just not have a `div` local variable, doing the `$(edit_form_html)` below instead.